File: Codeforces/C++/in_search_of_easy_problem.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    cin.tie(0);
    ios_base::sync_with_stdio(false);

    
    int k, n[100], i;
    int zero = 0;
    cin >> k;
    for (i = 0; i<k; i++) {
        scanf("%d", &n[i]);
    }
    if (k==1) {
        if (n[0] == 0) {
            printf("EASY");
        }
        if (n[0] == 1) {
            printf("HARD");
        }
    }

    else if(k > 1) {
        for (i = 0; i<k; i++) {
           if (n[i] == 1) {
               printf("HARD");
               break;
           }


          if (n[i] == 0) {
               zero += 1;

               if ( zero == k) {
                   printf("EASY");
                   break;
               }

          }
           
         }

   
    }

    


    return 0;

}

File: Codeforces/Python/in_search_of_easy_problem.py
k = int(input(''))
zero = 0
if k == 1:
    n = int(input(''))
    if (n == 1):
        print("HARD")

    if (n == 0):
        print("EASY")

if k > 1:
    n = []
    for i in range(k):
        n.append(int(input('')))

    for i in n:
        if i == 1:
            print("HARD")
            break
        if i == 0:
            zero += 1

            if (zero == len(n)):
                print("EASY")
